src/sentry/utils/outcomes.py

```diff
@@ -27,10 +27,16 @@ def api_name(self) -> str:
     def parse(cls, name: str) -> "Outcome":
         return Outcome[name.upper()]
 
+    def is_billing(self) -> bool:
+        return self in (Outcome.ACCEPTED, Outcome.RATE_LIMITED)
```

Comment:
I don't know much about int enum, can we trust it to do its job correctly? I wonder if this should be `self in (Outcome.ACCEPTED, Outcome.ACCEPTED.value, Outcome.RATE_LIMITED, Outcome.RATE_LIMITED.value)` just because

Reply:
That doesn't make sense to me. We would be comparing an instance of `self: Outcome` with an `int`. That _does_ work in fact, but I'd rather compare same types. I'll be adding tests for everything.

Reply:
Ah, I see now where this is coming from. While the code writes `ACCEPTED = 0`, in fact `intenum` makes it so that `Outcome.ACCEPTED` is of type `Outcome`:

```
>>> from sentry.utils.outcomes import Outcome
>>> Outcome.ACCEPTED
<Outcome.ACCEPTED: 0>
```

Reply:
Yeah thinking more about it, it'd crash beforehand because is_billing is not a method on int. you can probably do Outcome.is_billing(123) though